### ISSUE TITLE
Schema: projects.publications?

### ISSUE THREAD

**Issue by wolverdude** [peida2015/SciTwits-Proposal#5]
What is `projects.publications` for? It isn't clear to me upfront.

**Comment by peida2015** [comment]
It's a place for a research team to show a list of links to the publications related to the project.  I guess it might be best to make a new table for them?

**Comment by wolverdude** [comment]
Yes, that would be better. Then you could include article titles and journal titles along with the links.

This feels non-essential to me though. Perhaps you could add this in the bonus stage.
